scripts/fonts.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+FONT_VARIABLE="source-sans-3-latin-wght-normal.woff2"
+BASE_DIR="./static.in/cfgov-fonts/fonts"
+
+# Add required webfonts locally to docs/fonts/ directory.
+mkdir -p $BASE_DIR
+echo "Copying ./node_modules/@fontsource-variable/source-sans-3/files/$FONT_VARIABLE to $BASE_DIR/$FONT_VARIABLE"
+cp ./node_modules/@fontsource-variable/source-sans-3/files/$FONT_VARIABLE $BASE_DIR/$FONT_VARIABLE
```

Comment:
also need to add @fontsource-variable/source-sans-3 to package.json for this script to work

Reply:
Should no longer be necessary as of DS v4.0.1